### PR TITLE
Complete GUI

### PR DESCRIPTION
**Pull request by ngdao**
there is a main() in DiceGameGUI.java. I'm thinking of importing the classes that you have and create objects in this main. Don't know how we're gonna incorporate this

### CODE REVIEW

**Comment by anderspitman** [comment]
@hehe876 this one looks much better! Is it building ok as-is? It's easy to change gradle to point to DiceGameGUI.java, ie you don't have to use Main.java if it's not convenient. I don't want to hamper your workflow. If this is going to mess up using netbeans we can find an alternative.

**Comment by ngdao** [comment]
I put JAR file in the gui folder, the source code and pictures is gui/src. Changed DiceGame.java to my need. Have a look

File: exit [hunk 1]
```diff
@@ -0,0 +1,356 @@
+[33mcommit 25687869281ce9f41599e28d0a4a113cfc960482[m
```
Comment (anderspitman) [inline exit]: can you remove this file from git?

File: shared/src/main/java/DiceGame.java [hunk 2]
```diff
@@ -4,27 +4,22 @@
...
+    private int[] lastRoll;
     private static final int MAX_TOTAL = 23;
 
     public DiceGame() {
         currentTotal = 0;
+        lastRoll = new int[3];
         die = new Die();
     }
 
-    public int rollSingle() throws RollAfterGameOverException {
```
Comment (anderspitman) [inline shared/src/main/java/DiceGame.java:14]: This code is shared with the CLI. Removing it breaks the build.

File: shared/src/main/java/DiceGame.java [hunk 3]
```diff
@@ -4,27 +4,22 @@
 public class DiceGame {
     private Die die;
     private int currentTotal;
+    private int[] lastRoll;
```
Comment (anderspitman) [inline shared/src/main/java/DiceGame.java:7]: What is this used for?

**Comment by anderspitman** [comment]
@hehe876 it's best not to commit the jar directly to git. That can really slow things down. Usually only text files should be committed. I can add the jar to our releases. Can you remove it from your branch and push again?

**Comment by anderspitman** [comment]
Hm the jar's not working for me when I download it and run it directly.

`Exception in thread "main" java.lang.UnsupportedClassVersionError: DiceGameUI/DiceGameGUI : Unsupported major.minor version 52.0`

**Pull request closed without merging**